app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -110,7 +110,7 @@ private boolean delete(Media media, String reason) {
             mwApi.appendEdit(editToken, logPageString + "\n",
                     "Commons:Deletion_requests/" + date, summary);
             mwApi.appendEdit(editToken, userPageString + "\n",
-                    "User_Talk:" + sessionManager.getCurrentAccount().name, summary);
+                    "User_Talk:" + media.getCreator(), summary);
```

Comment:
Can you add/update the corresponding unit test

Reply:
@maskaravivek can you help me with unit test. i have never done a test. like what should be the result and where should it be placed .